homeassistant/components/pcal9535a/binary_sensor.py

```diff
@@ -0,0 +1,94 @@
+"""Support for binary sensor using I2C PCAL9535A chip."""
+import logging
+
+import voluptuous as vol
+
```

Comment:
Please remove this blank line. I recommend the `isort` tool to automatically sort and group imports.

Reply:
to myself:
```suggestion
```